presto-main/src/main/java/com/facebook/presto/execution/buffer/BroadcastOutputBuffer.java

```diff
@@ -69,6 +73,11 @@
     private final AtomicLong totalRowsAdded = new AtomicLong();
     private final AtomicLong totalBufferedPages = new AtomicLong();
 
+    private final ConcurrentMap<Lifespan, AtomicLong> outstandingPagePerLifespan = new ConcurrentHashMap<>();
```

Comment:
Looks like my comments above for `ArbitraryOutputBuffer.java` applies here as well.